pkg/local_object_storage/shard/get.go

```diff
@@ -140,3 +142,53 @@ func (s *Shard) getBytesWithMetadataLookup(addr oid.Address, skipMeta bool) ([]b
 	}
 	return b, err
 }
+
+// GetStream reads an object from shard as a stream. skipMeta flag allows to fetch object from
+// the blobstor directly.
+//
+// Returns the object header and a reader for the payload.
+// The caller is responsible for closing the returned io.ReadCloser if it is not nil.
+//
+// Returns any error encountered that did not allow to completely read the object part.
+// Returns an error of type apistatus.ObjectNotFound if the requested object is missing in shard.
+// Returns an error of type apistatus.ObjectAlreadyRemoved if the requested object has been marked as removed in shard.
+// Returns the object.ErrObjectIsExpired if the object is presented but already expired.
```

Comment:
`is present` (I know it's copypasted, but let's not reintroduce this error again)